### PR TITLE
More moods

### PR DESCRIPTION
**Pull request by ike709**
Everyone will be depressed all the time.

### CODE REVIEW

File: code/modules/mob/living/carbon/happiness_events/generic_events.dm [hunk 1]
```diff
@@ -21,10 +21,47 @@
 	happiness = 3
 	timeout = 3000
 
+/datum/happiness_event/book_nerd
+	description = "<span class='green'>I read a book.</span>\n"
```
Comment (Qustinnus) [inline code/modules/mob/living/carbon/happiness_events/generic_events.dm]: Make it a bit more descriptive, like "I have recently read a book"

File: code/modules/mob/living/carbon/happiness_events/generic_events.dm [hunk 2]
```diff
@@ -21,10 +21,47 @@
...
 	happiness = -8
 
+/datum/happiness_event/suffocation
+	description = "<span class='boldwarning'>CAN'T... BREATHE...</span>\n"
+	happiness = -6
+
+/datum/happiness_event/burnt_thumb
+	description = "<span class='warning'>I shouldn't play with lighters...</span>\n"
+	happiness = -1
+	timeout = 600
```
Comment (Qustinnus) [inline code/modules/mob/living/carbon/happiness_events/generic_events.dm]: make it like 2 min, its minor af anyway

File: code/modules/mob/living/carbon/happiness_events/generic_events.dm [hunk 3]
```diff
@@ -21,10 +21,47 @@
...
+/datum/happiness_event/cold
+	description = "<span class='warning'>It's way too cold in here.</span>\n"
+	happiness = -2
+
+/datum/happiness_event/hot
+	description = "<span class='warning'>It's getting hot in here.</span>\n"
+	happiness = -2
+
+/datum/happiness_event/creampie
+	description = "<span class='warning'>I've been creamed. Tastes like pie flavor.</span>\n"
```
Comment (Qustinnus) [inline code/modules/mob/living/carbon/happiness_events/generic_events.dm]: what the fuck is pie flavor 🤔

File: code/modules/mob/living/carbon/life.dm [hunk 4]
```diff
@@ -388,13 +390,17 @@
 	switch(bodytemperature)
```
Comment (Qustinnus) [inline code/modules/mob/living/carbon/life.dm]: You need to clear event on every occasion where the thing doesn't occur in the switch statement, else you will get edge cases where temperature changes violently and you go from cold to hot but you still have a cold moodlet.

File: code/modules/mob/living/carbon/happiness_events/generic_events.dm (hunk 3 above)
Comment (ike709) [reply code/modules/mob/living/carbon/happiness_events/generic_events.dm]: see discord

File: code/modules/mob/living/carbon/happiness_events/generic_events.dm (hunk 2 above)
Comment (ike709) [reply code/modules/mob/living/carbon/happiness_events/generic_events.dm]: I thought you were handling balance memes.

**Pull request merged**